docs/news.d/general.rst

```diff
@@ -0,0 +1,37 @@
+.. news-prs: 4298 4292 4262 4295 4226 4293 4253 4252 4250 4265 4254 4219 4216
```

Comment:
This fragment seems to collect changes to multiple aspects of OpenDDS. Should we break it up into smaller fragments?

Reply:
Yes, probably. Fragments, at least how I imagined them are a related set of changes.